src/helics/core/Core.hpp

```diff
@@ -848,9 +848,12 @@ class Core {
     @param queryFunction  a function object that returns a string as a result of a query in the form
     of const string ref. This callback will be called when a federate received a query that cannot
     be answered that directed at a particular federate
+    @param order multiple query callbacks can be set (0 or 1) is the first, then 2 and so on up to
```

Comment:
Docstring is a bit confusing -- is it some form of priority order that query callbacks should be run in if multiple query callbacks are set? Can multiple query callbacks be set with the same order value and what happens if you do? I think it should also mention that a value of 0 (or less) is equivalent to 1, and a value greater than 10 is equivalent to 10.

Reply:
I needed a way to handle multiple layers of callback queries.   Setting the same order just replaces any existing one, and I will clarify the docstring